flytekit/core/base_task.py

```diff
@@ -274,8 +274,11 @@ def local_execute(
             )
             outputs_literal_map = LocalTaskCache.get(self.name, self.metadata.cache_version, input_literal_map)
             # The cache returns None iff the key does not exist in the cache
-            if outputs_literal_map is None:
-                logger.info("Cache miss, task will be executed now")
+            if outputs_literal_map is None or local_config.cache_overwrite:
```

Comment:
if cache_overwrite is enabled, we still retrieve the output from LocalTaskCache. could we update the code to something like

```python
if cache_overwrite:
    self.sandbox_execute(...)
else:
    outputs_literal_map = LocalTaskCache.get(...)
    ...
```

Reply:
updated